tests/conftest.py

```diff
@@ -70,3 +73,361 @@ def _sample_all_wsis(_sample_ndpi, _sample_svs, _sample_jp2, tmpdir_factory):
         shutil.copy(_sample_jp2, dir_path.joinpath(_sample_jp2.basename))
 
     return dir_path
+
+
+@pytest.fixture(scope="session")
+def _source_image(tmpdir_factory):
+    """Sample pytest fixture for src image. Download src image image for pytest."""
+
+    src_image_file_path = tmpdir_factory.mktemp("data").join("source_image.png")
+
+    if not pathlib.Path(src_image_file_path).is_file():
+        print("\nDownloading Source Image")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata"
+                         "/common/source_image.png")
+        with open(src_image_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping Source Image")
+
+    return src_image_file_path
+
+
+@pytest.fixture(scope="session")
+def _norm_macenko(tmpdir_factory):
+    """Sample pytest fixture for norm_macenko image.
```

Comment:
"""Sample pytest fixture for norm_macenko image.
    Download norm_macenko image for pytest.

"""
Multiline docstring should have an empty line at the bottom.

Reply:
Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/pull/99/commits/f140ef4378c6618ae9d1f5cc7e47785c9c6da702